library/network/src/modules/NetworkService.rb

```diff
@@ -125,7 +125,7 @@ def backend_available?(backend)
       PackageSystem.Installed(BACKEND_PKG_NAMES[backend])
     end
 
-    alias_method :is_backend_available, :backend_available?
+    alias is_backend_available backend_available?
```

Comment:
to be honest, this is one of new cops which I really dislike ( I also disable it in my projects :) . Reason is that alias is very specific and easy to use incorrectly, very dangerous method. On other hand alias_method is common method, with reasonable parameters ( so not method names which looks like it should be called, but with symbol ) and also provide reasonable error if not found.

https://github.com/config-files-api/config_files_api/blob/master/.rubocop.yml#L55